site/en/docs/extensions/reference/storage/index.md

```diff
@@ -2,26 +2,66 @@
 api: storage
 ---
 
-## Overview
+## Overview {: #overview }
 
-This API has been optimized to meet the specific storage needs of extensions. It provides the same
-storage capabilities as the [localStorage API][local-storage] with the following key differences:
+The Storage API provides an extension-specific way to persist user data and state. It's similar to the web platform's storage APIs ([IndexedDB][mdn-indexeddb], [localStorage][mdn-localstorage]), but was designed to meet the storage needs of extensions. The following are a few key features: 
 
-- User data can be automatically synced with Chrome sync (using `storage.sync`).
-- Your extension's content scripts can directly access user data without the need for a background
-  page.
-- A user's extension settings can be persisted even when using [split incognito
-  behavior][incognito].
-- It's asynchronous with bulk read and write operations, and therefore faster than the blocking and
-  serial `localStorage API`.
-- User data can be stored as objects (the `localStorage API` stores data in strings).
-- Enterprise policies configured by the administrator for the extension can be read (using
-  `storage.managed` with a [schema][api-storage]).
+- Content scripts can directly access chrome.storage.
+- Stores JSON serializable values as object properties.
+- Storage is asynchronous with bulk read and write operations.
+- Storage data persists when the user clears the cache and browsing history.
+- Available in the service worker, content scripts, and other extension contexts.
+- A user's extension settings persist even when using [split incognito][incognito].
+- Offers an exclusive read-only [managed storage area][manifest-storage] for enterprise policies.
 
-## Manifest
+{% Details %}
+{% DetailsSummary %}
+💡 Can extensions use Window.localStorage?
+{% endDetailsSummary %}
 
-You must declare the "storage" permission in the [extension manifest][doc-manifest] to use the storage
-API. For example:
+Even though extensions can access [Window.localStorage][mdn-localstorage] in some contexts (popup and other HTML pages), it is not recommended for the following reasons:
+
+- The extension's service worker cannot access the localStorage API.
+- Content scripts share the localStorage of the host page.
+- Data saved to localStorage is lost when the user clears their browsing history.
+
+{% endDetails %}
+
+### Storage areas
+
+The Storage API is divided into the following four buckets ("storage areas"): 
+
+- [**storage.local**][prop-local]
```

Comment:
```suggestion
[`storage.local`][prop-local]
```

Reply:
Please repeat throughout the list.
* Try to avoid nested bullets or numbered lists.
* That this is a separate line and in a different font sets it off sufficiently enough to make the bullet superfluous.

I'm not sure I would have done bulleted lists here at all. The formatting for that is below (in case you don't know it). GitHub doesn't render this, but our site does.

[`storage.local`][prop-local]
: Stores data locally. The data is cleared when the extension is removed. The quota limitation...

Reply:
Gotcha 👍